cluster/gce/manifests/etcd.manifest

```diff
@@ -39,6 +39,9 @@
       },
       { "name": "ETCD_CREDS",
         "value": "{{ etcd_creds }}"
+      },
+      { "name": "ETCD_SNAPSHOT_COUNT",
+        "value": "10000"
```

Comment:
You didn't set this default on the kubeadm change.

Reply:
Added to `cmd/kubeadm/app/phases/etcd/local.go`